### ISSUE TITLE
Launch Game directly (Shortcut)

### ISSUE THREAD

**Issue by maximilianoe** [moonlight-stream/moonlight-chrome#301]
I'm only Streaming the whole desktop (via mstsc.exe) with one host. Is there a way to directly launch the "game" from the host with a shortcut? So not start -> select PC -> select game, just click a shortcut -> game launches on specific host.

I'm sorry if this has been asked before but I couldn't find anything.

**Comment by Jorys-Paulin** [comment]
Is this what you're thinking about? Sorry for the bad mockup.
![screen](https://user-images.githubusercontent.com/5084309/29914912-6dc2d75a-8e3a-11e7-916e-cc8ef813a796.png)

**Comment by maximilianoe** [comment]
Thanks for the answer but no, that's not what I meant.
Maybe this picture is better understandable?
![moonlightchromeshortcut](https://user-images.githubusercontent.com/15032541/29915624-a7da13b6-8e3c-11e7-9aba-066b8c4015c4.png)
I just want a shortcut to directly launch a specific host/game without interacting with the interface first.

**Comment by Jorys-Paulin** [comment]
@maximilianoe Yeah, we already discussed this in another issue, the idea was to use the URL parameters that you have sometimes in web pages, like `https://www.google.com/?gws_rd=ssl`, where `gws_rd` is passed to the web page as a parameter, which Moonlight can use for your request

**Comment by maximilianoe** [comment]
Okay, so if I'm understanding you correctly it's currently not possible to do this?

**Comment by Jorys-Paulin** [comment]
It would require implementation, which could be quite simple if the code weren't that messy.

**Comment by AnthraxLeprosyPi** [comment]
Hey there everybody, 

just stumbled upon Moonlight after years of performance issues with Steam Big Picture Mode and streaming...
Is there any update to this issue? Having a way to start games directly would be awesome. 
At the moment it's kind of cumbersome so have Kodi on my HTPC start Moonlight and than having to either use a mousse, teamviewer or mouse emulation on my XBOX360 controllers to start a game.

It just breaks the experience... 

best regards
Anthrax

**Comment by Jorys-Paulin** [comment]
@AnthraxLeprosyPi So you mean like a controller support? Controller support is kinda hard to implement here, as the code is pretty messy. It's not impossible though

**Comment by AnthraxLeprosyPi** [comment]
@Jorys-Paulin well ideally, I'd like to have a way to start games directly from within Kodi ;-)
Moonlight shortcuts with game launch parameters would be fine as there are Kodi plugins that can launch these...

**Comment by chrisism** [comment]
@Jorys-Paulin 
Any parameter support yet, as the original question was? I could really use it.

@AnthraxLeprosyPi 
Looking at Moonlight support in Kodi? I am busy with it, at the moment I only have the original nvidia client working on Android shield and getting the old Java implementation of moonlight working.

But I really need parameter support to get things going with Moonlight, else I will have to abandon all other OSes except the Shield/Android because that works fine on Kodi.

**Comment by Jorys-Paulin** [comment]
@chrisism I'd rather do it, but I find the code too messy to implement that in a clean way...

**Comment by schmurtzm** [comment]
Yes good idea to be able to run always the same game quickly from another computer.... Just click on the shortcut and go , don't mind on which machine it is hosted (yes a kind of dream ;) )
